### PR TITLE
ADEN-6668 Fix wrong selectors for n&s ad slots

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandom.java [hunk 1]
```diff
@@ -17,7 +17,6 @@
 import java.util.concurrent.TimeUnit;
 
 public class TestAdsVuapFandom extends AdsFandomTestTemplate {
-  private static final long MAX_MOVIE_DURATION = 40L;
   private static final int DESKTOP_VIDEO_TRIGGER_AREA = 2;
   private static final int MOBILE_VIDEO_TRIGGER_AREA = 3;
   private static final int REDIRECT_AREA_TRIGGER = 1; // DESKTOP & MOBILE
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandom.java]: MINOR Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsFandomContent.java#L29](https://github.com/Wikia/selenium-tests/blob/acbfc9e27d446cd7cd3068df9ffe90758da990a7/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L29): Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsFandomContent.java#L32](https://github.com/Wikia/selenium-tests/blob/acbfc9e27d446cd7cd3068df9ffe90758da990a7/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L32): Define a constant instead of duplicating this literal "div[id$='TOP_BOXAD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java [hunk 2]
```diff
@@ -21,21 +22,6 @@
   public static final String BOTTOM_BOXAD = "BOTTOM_BOXAD";
   public static final String FEED_BOXAD = "FEED_BOXAD";
 
-  static {
-    SLOT_SELECTORS = new HashMap<>();
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java]: I believe that if you restore this unfortunate map and fix the remaining 4 failing jobs, so they use selectors from this map all jobs will pass.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsFandomContent.java#L29](https://github.com/Wikia/selenium-tests/blob/137c99bd578633eb455f6484e07724c92aa3e65f/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L29): Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsFandomContent.java#L32](https://github.com/Wikia/selenium-tests/blob/137c99bd578633eb455f6484e07724c92aa3e65f/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L32): Define a constant instead of duplicating this literal "div[id$='TOP_BOXAD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsFandomContent.java#L29](https://github.com/Wikia/selenium-tests/blob/6367c3dc4546b7797f186f01b8a891111c926849/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L29): Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsFandomContent.java#L32](https://github.com/Wikia/selenium-tests/blob/6367c3dc4546b7797f186f01b8a891111c926849/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L32): Define a constant instead of duplicating this literal "div[id$='TOP_BOXAD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsFandomContent.java#L29](https://github.com/Wikia/selenium-tests/blob/137c99bd578633eb455f6484e07724c92aa3e65f/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L29): Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsFandomContent.java#L32](https://github.com/Wikia/selenium-tests/blob/137c99bd578633eb455f6484e07724c92aa3e65f/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L32): Define a constant instead of duplicating this literal "div[id$='TOP_BOXAD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsFandomContent.java#L29](https://github.com/Wikia/selenium-tests/blob/a83a82d884b58182a573d28aeee6759ce582a661/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L29): Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsFandomContent.java#L32](https://github.com/Wikia/selenium-tests/blob/a83a82d884b58182a573d28aeee6759ce582a661/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L32): Define a constant instead of duplicating this literal "div[id$='TOP_BOXAD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsFandomContent.java#L29](https://github.com/Wikia/selenium-tests/blob/7c805c30c9969320f253762b17d5cf653b223d9b/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L29): Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsFandomContent.java#L32](https://github.com/Wikia/selenium-tests/blob/7c805c30c9969320f253762b17d5cf653b223d9b/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L32): Define a constant instead of duplicating this literal "div[id$='TOP_BOXAD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by fraszczakszymon** [comment]
LGTM but https://github.com/Wikia/selenium-tests/pull/2166/files#diff-a990107b22280c3c37111e7e1847531cR45 this slot doesn't exist

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsFandomContent.java#L29](https://github.com/Wikia/selenium-tests/blob/130d5638f50baeb68753381120e0bd9de44b0c0c/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L29): Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsFandomContent.java#L32](https://github.com/Wikia/selenium-tests/blob/130d5638f50baeb68753381120e0bd9de44b0c0c/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L32): Define a constant instead of duplicating this literal "div[id$='TOP_BOXAD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsFandomContent.java#L29](https://github.com/Wikia/selenium-tests/blob/942cefc6d06c9e30f50a0e27a147282ae6274fa2/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L29): Define a constant instead of duplicating this literal "div[id$='BOTTOM_LEADERBOARD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsFandomContent.java#L32](https://github.com/Wikia/selenium-tests/blob/942cefc6d06c9e30f50a0e27a147282ae6274fa2/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java#L32): Define a constant instead of duplicating this literal "div[id$='TOP_BOXAD_0__container__']" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**